File: abc162/a/a.cpp
#include <iostream>

using namespace std;

bool luck(string n) {
  for (int i = 0; i < 3; i++) {
    if (n[i] == '7') {
      return true;
    }
  }
  return false;
}

int main(void) {
  string n;
  cin >> n;

  if (luck(n)) {
    cout << "Yes\n";
  } else {
    cout << "No\n";
  }
  return 0;
}

File: abc162/a/a.go
package main

import "fmt"

func luck(n string) bool {
	for i := 0; i < 3; i++ {
		if n[i] == '7' {
			return true
		}
	}
	return false
}

func main() {
	var n string
	fmt.Scan(&n)

	if luck(n) {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
